### PR TITLE
313758-Configured-the-EmbedProperties-by-EmbedConfig-Json

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,61 @@
-# BoldBI Embedding ASP.NET MVC Samples
+# Bold BI Embedded Sample in .NET MVC
```
Comment (vinothkumar1732) [inline README.md:1]: gitleaks.yml files is needed. It has been added by CI team. 
image

File: README.md [hunk 2]
```diff
@@ -1,64 +1,61 @@
...
- * [.NET Framework 4.8](https://dotnet.microsoft.com/en-us/download/dotnet-framework)
+ * [.NET Framework 4.8.1](https://dotnet.microsoft.com/en-us/download/dotnet-framework)
+
+ #### Help link
+
+ * https://help.boldbi.com/embedded-bi/faq/where-can-i-find-the-product-version/
+
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, and Safari.
```
Comment (vinothkumar1732) [inline README.md]: Remove the Safari as MVC not supporting in mac

File: README.md [hunk 3]
```diff
@@ -1,64 +1,61 @@
...
+
+    ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+ * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+
+    ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+    ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+ * Copy the downloaded embedConfig.json file and paste it into the designated [location](https://github.com/boldbi/aspnet-mvc-sample/tree/master/BoldBI.Embed.Sample) within the application. Please ensure that you have placed it in the application as shown in the following image.
```
Comment (vinothkumar1732) [inline README.md]: Highlight the embedConfig.json in the statement.

Comment (vinothkumar1732) [inline README.md]: Image is outdated. Please update with latest one.
image

File: README.md [hunk 4]
```diff
@@ -1,64 +1,61 @@
...
-        <td align="left">UserEmail of the Admin in your Bold BI, which would be used to get the dashboard list.</td>
-    </tr>
-  </tbody>
-</table>
-
-
-Please refer to the [help documentation](https://help.boldbi.com/embedded-bi/javascript-based/samples/v3.3.40-or-later/asp-net-mvc/#how-to-run-the-sample) to know how to run the sample.
-
-## Online Demos
+  * Open the solution file `BoldBI.Embed.Sample.sln` in Visual studio.
```
Comment (vinothkumar1732) [inline README.md]: Open the solution file `BoldBI.Embed.Sample.sln` in Visual Studio.

**Review by vinothkumar1732 — COMMENTED** [review]
gitleaks.yml files is needed. It has been added by CI team.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:1]: Okay, now I have included.

File: README.md (hunk 4 above)
Comment (ghost) [reply README.md]: Content updated

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: Latest image updated

Comment (ghost) [reply README.md]: embedConfig.json statement highlighted

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: yes, removed

File: BoldBI.Embed.Sample/BoldBI.Embed.Sample.csproj [hunk 5]
```diff
@@ -1,239 +1,267 @@
...
+      </FlavorProperties>
+    </VisualStudio>
+  </ProjectExtensions>
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1\build\net46\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.2.0.1\build\net46\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props'))" />
+  </Target>
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
```
Comment (florenceliliana) [inline BoldBI.Embed.Sample/BoldBI.Embed.Sample.csproj:262]: <!-- To modify your build process, add your task inside one of the targets in the following and uncomment it.

File: BoldBI.Embed.Sample/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -1,35 +1,35 @@
...
+[assembly: AssemblyCopyright("Copyright ©  2023")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
```
Comment (florenceliliana) [inline BoldBI.Embed.Sample/Properties/AssemblyInfo.cs:22]: // The following GUIDE is for the ID of the typelib if this project is exposed to COM

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: BoldBI.Embed.Sample/Properties/AssemblyInfo.cs (hunk 6 above)
Comment (ghost) [reply BoldBI.Embed.Sample/Properties/AssemblyInfo.cs:22]: Content updated

File: BoldBI.Embed.Sample/BoldBI.Embed.Sample.csproj (hunk 5 above)
Comment (ghost) [reply BoldBI.Embed.Sample/BoldBI.Embed.Sample.csproj:262]: Content updated

**Review by florenceliliana — APPROVED** [review]

**Review by Alagarsamyd — APPROVED** [review]

**Pull request merged**